server/src/main/java/org/elasticsearch/cluster/ClusterInfo.java

```diff
@@ -204,8 +216,8 @@ public Iterator<? extends ToXContent> toXContentChunked(ToXContent.Params params
                 return builder.endObject(); // NodeAndPath
             }),
             endArray() // end "reserved_sizes"
-            // NOTE: We don't serialize estimatedHeapUsages at this stage, to avoid
-            // committing to API payloads until the feature is settled
+            // NOTE: We don't serialize estimatedHeapUsages/nodeWriteLoads at this stage, to avoid
+            // committing to API payloads until the features are settled
```

Comment:
I'm not entirely sure what the toXContentChunked is used for, but figured I should follow suit.